packages/contracts-bedrock/.storage-layout

```diff
@@ -6,26 +6,27 @@
-| PORTAL             | contract OptimismPortal  | 249  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| Name               | Type                      | Slot | Offset | Bytes | Contract                                                 |
+|--------------------|---------------------------|------|--------|-------|----------------------------------------------------------|
+| spacer_0_0_20      | address                   | 0    | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| _initialized       | uint8                     | 0    | 20     | 1     | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| _initializing      | bool                      | 0    | 21     | 1     | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_1_0_1600    | uint256[50]               | 1    | 0      | 1600  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_51_0_20     | address                   | 51   | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_52_0_1568   | uint256[49]               | 52   | 0      | 1568  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_101_0_1     | bool                      | 101  | 0      | 1     | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_102_0_1568  | uint256[49]               | 102  | 0      | 1568  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_151_0_32    | uint256                   | 151  | 0      | 32    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_152_0_1568  | uint256[49]               | 152  | 0      | 1568  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_201_0_32    | mapping(bytes32 => bool)  | 201  | 0      | 32    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| spacer_202_0_32    | mapping(bytes32 => bool)  | 202  | 0      | 32    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| successfulMessages | mapping(bytes32 => bool)  | 203  | 0      | 32    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| xDomainMsgSender   | address                   | 204  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| msgNonce           | uint240                   | 205  | 0      | 30    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| failedMessages     | mapping(bytes32 => bool)  | 206  | 0      | 32    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| __gap              | uint256[42]               | 207  | 0      | 1344  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| PORTAL             | contract OptimismPortal   | 249  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| superchainConfig   | contract SuperchainConfig | 250  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
```

Comment:
big diff due to the column width change. The only real diff is the addition of the new superchainConfig var at the end